Assessments.Frontend.Web/wwwroot/json/glossary.json

```
@@ -36,11 +36,16 @@
     "tagline": "Vurdert for Svalbard",
     "description": "Vurderingen gjelder for Svalbard, dvs. Spitsbergen og øyene omkring, samt Bjørnøya og Hopen inklusiv kyst innen 12 nautiske mil."
   },
+  "populationsize": {
+    "title": "Populasjonsandel",
+    "description": "Norge er ikke nødvendigvis alene om å ha en art. Men bestanden i Norge utgjør noen ganger en betydelig andel av de totale bestanden, som igjen kan påvirke rødlistestatusen.",
```

En liten skrivefeil.
```suggestion
    "description": "Norge er ikke nødvendigvis alene om å ha en art. Men bestanden i Norge utgjør noen ganger en betydelig andel av den totale bestanden, som igjen kan påvirke rødlistestatusen.",
```